locale/de/LC_MESSAGES/django.po

```diff
@@ -1903,21 +1902,23 @@ msgstr "Projekt auf meinBerlin"
 #, python-format
 msgid ""
 "\n"
-"                    Participation possible until %(date)s\n"
+"                    Participation possible until <time datetime=\"%(isodate)s"
+"\">%(date)s</time>\n"
 "                "
 msgstr ""
 "\n"
-"Beteiligung möglich bis zum %(date)s"
+"Beteiligung möglich bis zum <time datetime=\"%(isodate)s\">%(date)s</time>"
```

Comment:
Would it be possible to leave the string as-is and just insert the complete HTML?